components/WeeklyPickButton/WeeklyPickButton.test.tsx

```diff
@@ -3,19 +3,93 @@ import { render, screen } from '@testing-library/react';
 import { WeeklyPickButton } from './WeeklyPickButton';
 import { RadioGroup } from '../RadioGroup/RadioGroup';
 
+// test one method
+// provide some specific arguments to that method
+// test that the result is as expected
+
+const weeklyPickData = {
+  team: 'Ravens',
+  src: '/path/to/ravens.svg',
+  height: '48',
+  width: '48',
+  alt: 'Ravens',
+};
+
 describe('WeeklyPickButton', () => {
-  it('renders correctly', () => {
+  it('RadioGroup has the correct classes', () => {
+    render(<RadioGroup />);
+
+    const radioGroup = screen.getByRole('radiogroup');
+    expect(radioGroup).toHaveClass('grid w-full grid-cols-2 gap-4');
+  });
+
+  it('radio button has the correct classes', () => {
     render(
       <RadioGroup>
-        <WeeklyPickButton team="Test Team" src="/path/to/image.svg"/>
+        <WeeklyPickButton team={weeklyPickData.team} src={weeklyPickData.src} />
       </RadioGroup>,
     );
 
-    const image = screen.getByRole('img', { name: /Test Team/i });
-    expect(image).toBeInTheDocument();
-    expect(image).toHaveAttribute('src', '/path/to/image.svg');
+    const radioButton = screen.getByRole('radio');
+    expect(radioButton).toHaveClass('peer sr-only');
+  });
+
+  it('image should have a height of 48', () => {
+    render(
+      <RadioGroup>
```

Comment:
m: Remove RadioGroup. You need to test just the component for this file.

Reply:
When I remove RadioGroup, I recieve the following error and the test automatically fails:
```
RadioGroupItem` must be used within `RadioGroup
```